Platforms/QemuQ35Pkg/QemuQ35PkgX64.dsc

```diff
@@ -0,0 +1,1559 @@
+## @file
+#  Open Virtual Machine Firmware for the Q35 platform using Project Mu
+#
+#  Copyright (c) Microsoft Corporation
+#  Copyright (c) 2006 - 2019, Intel Corporation. All rights reserved.<BR>
+#  (C) Copyright 2016 Hewlett Packard Enterprise Development LP<BR>
+#
+#  SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+##
+
+################################################################################
+#
+# Defines Section - statements that will be processed to create a Makefile.
+#
+################################################################################
+[Defines]
+  PLATFORM_NAME                  = QemuQ35
+  PLATFORM_GUID                  = 163b507c-8702-496a-99d7-566c36e14728
+  PLATFORM_VERSION               = 0.1
+  DSC_SPECIFICATION              = 0x00010005
+  OUTPUT_DIRECTORY               = Build/QemuQ35Pkg

```

Comment:
It'd be nice to output this to a unique directory.

```suggestion
  OUTPUT_DIRECTORY               = Build/QemuQ35PkgX64
```

Reply:
Updated in the latest push